internal/transports/p2p/server.go

```diff
@@ -36,11 +35,28 @@ func NewServer(
 		headersService: headersService,
 		chainService:   chainService,
 		log:            &serverLogger,
+		peers:          make([]*peer.Peer, 0),
 	}
 	return server
 }
 
 func (s *server) Start() error {
+	err := s.seedAndConnect()
+	if err != nil {
+		return err
+	}
+
+	return s.listenAndConnect()
+}
+
+func (s *server) Shutdown() error {
+	for _, p := range s.peers {
+		_ = p.Disconnect()
```

Comment:
Maybe we should at least log the error instead of ignoring it 🤔 ?

Reply:
We're logging the error in `p.Disconnect()` method.

My idea for this implementation is to remove returning error from this method completely, and wait for Disconnection and/or error from all peers with WaitGroup in server shutdown. But this synchronisation will be done in the next task regarding multiple connections, while not returning the error can only be done after we decide to switch entirely to this solution and remove the interface for P2P that we temporarily created in the `main` method for this to work.

Reply:
Actually this comment bring some additional thoughts to me,
I checked the code, and there is no place where we're actually handling the error from Disconnect method.
And I have some doubts:
1. How actually we should handle such an error, 
2. is log message enough? or we're able to do something more?
3. if log message (or internal peer things) are enough, then we shouldn't actually return anything (or at least we shouldn't return error type) because this is making the code only missleading.

Reply:
Removed